boot_node/src/config.rs

```diff
@@ -105,6 +110,12 @@ impl<T: EthSpec> BootNodeConfig<T> {
 
             let mut local_enr = {
                 let mut builder = create_enr_builder_from_config(&network_config, false);
+                // Include the ipv6 ports too.
+                // NOTE: we do this explicitely to be able to upgrade the bootnodes to ipv6 without
+                // upgrading the beacon node yet.
+                if let Some(udp_port) = network_config.enr_udp_port {
+                    builder.udp6(udp_port);
+                }
```

Comment:
This means that every bootnode will advertise a udp6 port? And even if we set an ipv4 address in the config, we still create an IPv6 socket right? (There's prob no harm in doing this, just double checking)

I was thinking that we were going to change the `create_enr_builder_from_config()` and if an ipv6 address is given, then we do dual stack and advertise both ips and ports. 

I see that we want to take the cautious approach and not modify the beacon node in any way, but I think we use regular beacon nodes for bootnodes in the testnets.

What are your thoughts on extending this to include beacon nodes if an ipv6 address is given in the CLI config. We force everyone to use dual-stack initially I think if we went down that path?

Reply:
I would prefer if we update the boot nodes first tbh. This way we can gauge how ipv6 is working on live discv6 servers and continue from there